src/mx/Domain.java

```diff
@@ -0,0 +1,25 @@
+package mx;
+
+public class Domain {
+
+    protected String url;
+    protected Integer status;
```

Comment:
Those two fields should be private and final - prefer immutable objects, this would make for a better design. Make them required constructor parameters - such status object should be created in complete, valid state (containing a valid URL and integer status). I would also change `Integer` to `int` to stress that this field is required and cannot be `null`